admin/spec/controllers/service-manager.js

```diff
@@ -0,0 +1,270 @@
+/* global jasmine describe expect it beforeEach afterEach spyOn */
+const httpMocks = require('node-mocks-http')
+const sinon = require('sinon')
+const controller = require('../../controllers/service-manager')
+const settingService = require('../../services/setting.service')
+const checkWindowService = require('../../services/check-window.service')
+const settingsValidator = require('../../lib/validator/settings-validator')
+const checkWindowValidator = require('../../lib/validator/check-window-validator')
+const ValidationError = require('../../lib/validation-error')
+
+describe('service manager controller:', () => {
+  let sandbox
+  let next
+  function getRes () {
+    const res = httpMocks.createResponse()
+    res.locals = {}
+    return res
+  }
+
+  function getReq (params) {
+    const req = httpMocks.createRequest(params)
+    req.breadcrumbs = jasmine.createSpy('breadcrumbs')
+    req.flash = jasmine.createSpy('flash')
+    return req
+  }
+
+  beforeEach(() => {
+    sandbox = sinon.sandbox.create()
```

Comment:
I'm not sure why the sandbox is here?  I suspect it can be removed and a straight Jasmine implementation will work.

Reply:
good point.  pure jasmine definitely preferred going forward

Reply:
@GuyHarwood @jon-shipley it is not used at all, it was leftover code.
Removed